lms/static/js/instructor_dashboard/cohort_management.js

```diff
@@ -0,0 +1,29 @@
+(function() {
+  var CohortManagement;
+
+  CohortManagement = (function() {
+
+    function CohortManagement($section) {
+      this.$section = $section;
+      this.$section.data('wrapper', this);
+    }
+
+    CohortManagement.prototype.onClickTitle = function() {};
+
+    return CohortManagement;
+
+  })();
+
+  _.defaults(window, {
```

Comment:
This logic is odd, but I see that it is boilerplate generated by CoffeeScript in each of the instructor dashboard tabs. I wonder if there's a cleaner way to do this in pure JS.